docs/specification.md

```diff
@@ -93,6 +95,45 @@ The "Sign In" page includes:
 - "Password" field.
 - "Log In" button: Submits the form and redirects the user to the "Projects" page.
 
+Hide/Show password toggle must be located on the right side of a Password input field. Single-click to expose/hide password. (For both SignIn and SignUp)
+
+Red validation message (‘Field cannot be empty’) appears below the inputs when fields are empty. (for both SignIn, SignUp including Name, Email and Password fields).
```

Comment:
It should be a common clarification for forms in the app, not for the auth only. It should state that when form fields are invalid after submission attempt, the appropriate validation messages should be shown near the corresponding fields.
```suggestion
```